include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -406,7 +406,7 @@ class edit_distance_unbanded_semi_global_policy :
     size_t best_score_column() const noexcept
     {
         derived_t const * self = static_cast<derived_t const *>(this);
-        return std::ranges::distance(std::ranges::begin(self->database), _best_score_col);
+        return std::ranges::distance(std::ranges::begin(self->database), _best_score_col) + 1u;
```

Comment:
That means `_best_score_col` is still the column of the inclusive back coordinate, right?
But you define `best_score_column()` as the column of the exclusive back coordinate.

I suggest you either let `_best_score_col` contain the exclusive coordinate value as well or rename it to distinguish the different meanings.

Reply:
thank you, that's a good idea. I'll name the function `back_coordinate_column`